src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -525,6 +527,27 @@ internal Collection<FunctionDescriptor> ReadFunctions(List<FunctionMetadata> met
             return functionDescriptors;
         }
 
+        private List<FunctionMetadata> ApplyMaximumFunctionCount(List<FunctionMetadata> metadatas)
```

Comment:
Agree! `metadatas` was bugging me, but was already there, so I didn't want to change :) I'll update it throughout.